src/components/dev-hub/section-header.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { colorMap, fontSize, screenSize } from './theme';
+import { P } from './text';
+
+const SectionHeader = styled('div')`
+    border-bottom: 1px solid ${colorMap.greyDarkOne};
+    font-size: ${fontSize.small};
+    letter-spacing: 2px;
+    @media ${screenSize.upToMedium} {
+        display: block;
+    }
+    text-transform: uppercase;
+`;
+
+export default ({ text }) => (
+    <SectionHeader>
+        <P>{text}</P>
```

Comment:
```suggestion
        <P collapse>{text}</P>
```

Using the `collapse` prop will bring the border closer by removing the bottom margin on the `P` we decide to use.

Reply:
The design seems to have bottom margin separating the text and the border

Reply:
Ah yes there is `8px` between the text and the border. The updated `line-height` should give us this difference but if not good call let's keep that border. The gap seemed a bit high before but updating the text properties should help us.

Reply:
I left `collapse` out for now. Let me know what you think and I can change it if anything

Reply:
Okay cool, yeah I think the default margin on `P` is `16px` so let's also add that to the `styled` and the font size should be `12px`. Once those are in I think this should be good to go